pinot-core/src/main/java/org/apache/pinot/core/common/datatable/DataTableUtils.java

```diff
@@ -233,4 +243,98 @@ private static DataTable buildEmptyDataTableForDistinctQuery(QueryContext queryC
     dataTableBuilder.finishRow();
     return dataTableBuilder.build();
   }
+
+  /**
```

Comment:
These functions should also be moved to base class.

Reply:
done